### DISCUSSION TITLE
Subscriptions page to sponsor projects no exist

### DISCUSSION THREAD

**Discussion by bcastro-mcm** [capawesome-team/capacitor-plugins#286] [Q&A]
The Subscriptions page to sponsor plugins is currently unavailable.

the link is: https://polar.sh/capawesome-team/

![Captura de pantalla 2024-09-04 a la(s) 3 13 18 p  m](https://github.com/user-attachments/assets/e60cc668-69a6-4b30-838f-ed98a36e107d)

**Comment by robingenz** [comment]
Thank you for reporting this. It seems that the Polar team changed the URL and forgot to add a redirect. The new url is: https://polar.sh/capawesome-team/
